llvm/test/Transforms/ARMWidenStrings/arm-widen-strings-1.ll

```diff
@@ -0,0 +1,25 @@
+; RUN: opt < %s -mtriple=arm-arm-none-eabi -passes="default<O2>,globalopt" -S | FileCheck %s
```

Comment:
It should be enough to test with -passes=globalopt, and shouldn't need the whole pass pipeline.

Reply:
Done